### PR TITLE
Add steps to fake kernel release

### PR DESCRIPTION
**Pull request by pkratoch**
Separated from PR https://github.com/rpm-software-management/ci-dnf-stack/pull/688 so that it can be merged sooner.

### CODE REVIEW

**Comment by lukash** [comment]
LGTM, I'll just defer to @m-blaha for the changes to the Dockerfiles, as I'm not entirely sure they won't break anyhting.

File: dnf-behave-tests/features/steps/cmd.py [hunk 1]
```diff
@@ -71,6 +71,30 @@ def faketime_today(context, when):
     context.execute_steps('when I move the clock backward to "{}"'.format(when))
 
 
+@behave.step("I fake kernel release to {release}")
+def i_fake_kernel_release(context, release):
+    """Override uname() system call to return {release} as release.
+
+    Libdnf determines running kernel as an RPM package that provides the following file:
+        "/boot/vmlinuz-" + uname.release
+    By faking uname.release, all that remains to create fake running-kernel package is for the
```
Comment (j-mracek) [inline dnf-behave-tests/features/steps/cmd.py]: Please could you add note that @no_installroot and @destructive marks are needed for scenario where fake uname is used?

Comment (lukash) [reply dnf-behave-tests/features/steps/cmd.py]: The step doesn't do anything destructive itself...

I think the need for `@no_installroot` comes from the file needing to be present on the host? This is probably what should be mentioned here.

I also think this:
> Libdnf determines running kernel as an RPM package that provides the following file:
>    "/boot/vmlinuz-" + uname.release

Should be extended by mentioning that dnf does the `uname` syscall and uses the result to look for the file (the `uname.release` above), to make it more clear what exactly is going on.

Comment (j-mracek) [reply dnf-behave-tests/features/steps/cmd.py]: fake uname does not work with installroot (libdnf implementation), therefore you need to use @no_installroot

Comment (pkratoch) [reply dnf-behave-tests/features/steps/cmd.py]: Thank you both for feedback. ``@no_installroot`` is among the destructive tags by default. But I added a note that ``@no_installroot`` is necessary to fake running kernel. I also tried to clarify how libdnf checks for running kernel, so please review if it's understandable.

Comment (lukash) [reply dnf-behave-tests/features/steps/cmd.py]: I think it's clear now :+1:

**Comment by j-mracek** [comment]
LGTM

**Pull request merged**